src/OpenTelemetry.SemanticConventions/OpenTelemetry.SemanticConventions.csproj

```diff
@@ -0,0 +1,9 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
we can do net462 and higher only.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Api/OpenTelemetry.Api.csproj#L4

Reply:
@cijothomas  sorry I didn't have much time to look into these. Busy recent days.. could you please re-open this? I will try to get around this on the upcoming days/weekend.

Reply:
Done. Targeting now `net462`, updated publicApis.